### PR TITLE
Enable Talkman in Deployer 3

### PR DESCRIPTION
**Pull request by SuperJMN**

### CODE REVIEW

**Comment by SuperJMN** [comment]
You could take Cityman's script to compare. It only has a few changes. 
I assumed there are no difference between Dual and Single SIM devices.

File: Devices/Lumia/950s/Talkman/Main.txt [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+	BcdEdit(efiBcdStore, "/displayorder {{{bcdWinMobile}}} /addlast");
+	BcdEdit(efiBcdStore, "/default {{{bcdWoa}}}");
+	BcdEdit(efiBcdStore, "/timeout 30");
+}
+
+AddDrivers
+{
+	'Downloading drivers'
+	Unzip(GitHubLatestReleaseAsset("WOA-Project", "Lumia-Drivers", "HARDWARE.INPUT.SYNAPTICS_RMI4_F12_WIN10.zip"), packages);
+	Unzip(GitHubLatestReleaseAsset("WOA-Project", "Lumia-Drivers", "DEVICE.SOC_QC899X.CITYMAN.zip"), packages);
```
Comment (gus33000) [inline Devices/Lumia/950s/Talkman/Main.txt]: Should be DEVICE.SOC_QC899X.TALKMAN.zip instead, you probably missed that one :)

File: Devices/Lumia/950s/Talkman/Main.txt [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+	Unzip(GitHubLatestReleaseAsset("WOA-Project", "Lumia-Drivers", "DEVICE.SOC_QC899X.CITYMAN_DESKTOP.zip"), packages);
```
Comment (gus33000) [inline Devices/Lumia/950s/Talkman/Main.txt]: Same here as above

File: Devices/Lumia/950s/Talkman/Main.txt [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+	Unzip(GitHubLatestReleaseAsset("WOA-Project", "Lumia-Drivers", "SUPPORT.DESKTOP.EXTRAS.zip"), packages);
+	Unzip(GitHubLatestReleaseAsset("WOA-Project", "Lumia-Drivers", "SUPPORT.DESKTOP.MMO_EXTRAS.zip"), packages);
+	Unzip(GitHubLatestReleaseAsset("WOA-Project", "Lumia-Drivers", "SUPPORT.DESKTOP.MOBILE_BRIDGE.zip"), packages);
+	Unzip(GitHubLatestReleaseAsset("WOA-Project", "Lumia-Drivers", "SUPPORT.DESKTOP.MOBILE_COMPONENTS.zip"), packages);
+	Unzip(GitHubLatestReleaseAsset("WOA-Project", "Lumia-Drivers", "SUPPORT.DESKTOP.MOBILE_RIL.zip"), packages);
+	Unzip(GitHubLatestReleaseAsset("WOA-Project", "Lumia-Drivers", "SUPPORT.DESKTOP.MOBILE_RIL_EXTRAS.zip"), packages);
+
+	'Injecting Drivers'
+	InjectDrivers(windowsRoot, "{packages}\HARDWARE.INPUT.SYNAPTICS_RMI4_F12_WIN10");
+	InjectDrivers(windowsRoot, "{packages}\DEVICE.SOC_QC899X.CITYMAN");
```
Comment (gus33000) [inline Devices/Lumia/950s/Talkman/Main.txt]: Same problem here

File: Devices/Lumia/950s/Talkman/Main.txt [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+	InjectDrivers(windowsRoot, "{packages}\DEVICE.SOC_QC899X.CITYMAN_DESKTOP");
```
Comment (gus33000) [inline Devices/Lumia/950s/Talkman/Main.txt]: and here

File: Devices/Lumia/950s/Talkman/Main.txt [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+	InjectDrivers(windowsRoot, "{packages}\SUPPORT.DESKTOP.MOBILE_RIL");
+	InjectDrivers(windowsRoot, "{packages}\SUPPORT.DESKTOP.MOBILE_RIL_EXTRAS");
+}
+
+InstallUefi
+{
+	'Installing UEFI'
+	
+	UnzipRoot(GitHubLatestReleaseAsset("WOA-Project", "Lumia950XLPkg", "MSM8994.UEFI.Lumia.950.zip"), packages);
+	Copy("{packages}\MSM8994.UEFI.Lumia.950\ELF\UEFI.elf", "{efiEspRoot}\");
```
Comment (gus33000) [inline Devices/Lumia/950s/Talkman/Main.txt]: Should be MSM8992.UEFI.Lumia.950

File: Devices/Lumia/950s/Talkman/Main.txt [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+	InjectDrivers(windowsRoot, "{packages}\SUPPORT.DESKTOP.MOBILE_COMPONENTS");
+	InjectDrivers(windowsRoot, "{packages}\SUPPORT.DESKTOP.MOBILE_RIL");
+	InjectDrivers(windowsRoot, "{packages}\SUPPORT.DESKTOP.MOBILE_RIL_EXTRAS");
+}
+
+InstallUefi
+{
+	'Installing UEFI'
+	
+	UnzipRoot(GitHubLatestReleaseAsset("WOA-Project", "Lumia950XLPkg", "MSM8994.UEFI.Lumia.950.zip"), packages);
```
Comment (gus33000) [inline Devices/Lumia/950s/Talkman/Main.txt]: Should be MSM8992.UEFI.Lumia.950.zip here as well

**Review by gus33000 — CHANGES_REQUESTED** [review]
Hey @SuperJMN finished doing the review, looks like you forgot a few things to replace but nothing much, with the suggested changes it looks good to go for me :shipit:

**Comment by gus33000** [comment]
> You could take Cityman's script to compare. It only has a few changes. I assumed there are no difference between Dual and Single SIM devices.

You're right there's no difference these days, before we had different settings but it's all dynamic now

File: Devices/Lumia/950s/Talkman/Main.txt [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+
+ConfigureBcd
+{
+	'Configuring Boot'
+	
+	bcdWoa = "7619dcca-fafe-11d9-b411-000476eba25f";
+	bcdWinMobile = "7619dcc9-fafe-11d9-b411-000476eba25f";
+	efiBcdStore = "{efiEspRoot}\EFI\Microsoft\Boot\BCD";
+	
+	BcdEditCreate(efiBcdStore, bcdWoa, "/d ""Windows 10"" /application BOOTAPP");
```
Comment (gus33000) [inline Devices/Lumia/950s/Talkman/Main.txt:110]: (Nitpick) Would probably change this to "Windows Desktop" or "Full Windows" or "Windows on ARM" considering Windows 11 and other things)

**Review by gus33000 — APPROVED** [review]
Hey just a tiny nitpick but that's about it.

:shipit:

File: Devices/Lumia/950s/Talkman/Main.txt (hunk 7 above)
Comment (SuperJMN) [reply Devices/Lumia/950s/Talkman/Main.txt:110]: I'm make this change in the next PR, thank you mate!

**Pull request merged**